doc/conf.py

```diff
@@ -88,6 +88,13 @@ def _check_deps():
 
 autodoc_docstring_signature = True
 
+intersphinx_mapping = {
+  'python': ('https://docs.python.org/', None),
+  'numpy': ('https://docs.scipy.org/doc/numpy/', None),
+  'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
+  'pandas': ('https://pandas.pydata.org/pandas-docs/stable', None)
```

Comment:
This one's not accessible over https, unfortunately. Also, why two-space indent?